src/mam4xx/mam4_amicphys.hpp

```diff
@@ -2119,7 +2120,13 @@ void modal_aero_amicphys_intr(
     const Real pmid, const Real pdel, const Real zm, const Real pblh,
     const Real qv, const Real cld,
     // in/out
-    Real qq[gas_pcnst], Real qqcw[gas_pcnst],
+    Real (&qq)[gas_pcnst], Real (&qqcw)[gas_pcnst],
+    // Diagnostics (out)
+    Real (&gas_aero_exchange_condensation)[gas_pcnst],
+    Real (&gas_aero_exchange_renaming)[gas_pcnst],
+    Real (&gas_aero_exchange_nucleation)[gas_pcnst],
+    Real (&gas_aero_exchange_coagulation)[gas_pcnst],
+    Real (&gas_aero_exchange_renaming_cw)[gas_pcnst],
```

Comment:
We can remove `gas_aero_exchange_renaming_cw` for now until the eval team requests this field.